src/app/core/entity-components/historical-data/historical-entity-data.ts

```diff
@@ -0,0 +1,9 @@
+import { Entity } from "../../entity/entity";
+import { DatabaseEntity } from "../../entity/database-entity.decorator";
+import { DatabaseField } from "../../entity/database-field.decorator";
+
+@DatabaseEntity("HistoricalEntityData")
```

Comment:
JSDoc comments with a brief description of the purpose of the class and each properties